### PR TITLE
Merge

### PR DESCRIPTION
**Pull request by rkhamis**
Merging development into development_960

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -2,6 +2,7 @@ language: python
 sudo: enabled
 dist: trusty
```
Comment (xmonader) [inline .travis.yml]: is that the latest one? I thought jumpscale is going for 18.04 no?

Comment (rkhamis) [reply .travis.yml]: true dat! nice catch, yo

Comment (rkhamis) [reply .travis.yml]: seems like it's not yet supported by travis. Guess all the tests will fail immediately

Comment (zaibon) [reply .travis.yml]: This Travis doesnt have that version indeed. That's why the test script build a docker images of Ubuntu 18.04 and run the test inside

Comment (zaibon) [reply .travis.yml]: Woops miss clicked to resolved... Sorry

Comment (zaibon) [reply .travis.yml]: Woops miss clicked on resolved...
also I see the we don't use docker for the test. So yeah well have to check how to solve this. Maybe Juste using python 3.6 is enough

File: cmds/js9 [hunk 2]
```diff
@@ -0,0 +1 @@
+js_shell
```
Comment (zaibon) [inline cmds/js9]: This file can probably be removed

**Review by zaibon — CHANGES_REQUESTED** [review]
Need to review the templates files to make sure the autoformatting didn't screw them up.

Also during such a big PR. Mixing merge and auto formatting make it nearly impossible to review. Next time make a commit for the merge and one for the formatting.

**Review by zaibon — CHANGES_REQUESTED** [review]
Can we also remove the filtering of travis on the branches ?
I want to have travis running test for any branches

**Pull request closed without merging**